worker/worker.go

```diff
@@ -29,6 +31,24 @@ type Worker struct {
 	QueueId        string
 }
 
+type TaskState uint8
+
+const (
+	TaskPending TaskState = iota
+	TaskRunning
+	TaskFinished
+	TaskFailed
+)
+
+type Task struct {
+	Commands       []string
+	ReportInterval int64
+	State          TaskState
+	Progress       int
```

Comment:
add internal doc to explain the range of progress var. is it between 0 and 100?

Reply:
sure, my bad.

Reply:
rename to something like `indication of task completion progress, ranging from 0 to 100`

Reply:
done